docs/support/support_system.md

```diff
@@ -0,0 +1,164 @@
+
+##### Priority field
+A priority level such as P1, P2, P3, P4 etc that describes the urgentness of the ticket and the number of users it’s affecting. It’s good to have a team-wide common definition on what each priority level means and document it in a relevant place for everyone’s reference. Below is a suggested priority level based on Dimagi’s support process.
+- **P1** : Severe (a blocker), don't do anything else. May have to sleep less tonight. There is (business loss) already. The longer it's not fixed, the longer the product and the team are in failure state. Examples: Site down, data loss, security breakdown etc.
+- **P2** : A problem which has made an important/critical function unusable or unavailable and no workaround exists. Examples: All users not being able to sync with server.
+- **P3** : High (Should be fixed), if not fixed, will lose integrity in product. Example: Pillows falling behind by a large backlog.
+- **P4** : Normal (Should be fixed, time and resources permitting)
+- **P5** : Low
+
+The priority level helps the entire support team and developers to understand how they should prioritize the particular ticket. A support team member triaging the ticket can setup the priority.
+
+##### Ticket Workflow
+Once the support system is set up below is a general process that can be followed. Note that for P1/P2 we recommend a separate on-call like process stated in the [P1/P2 process](#p1-p2-process) recommendations.
+
+<p align="center">
+    <img src="local_hosting_support_workflow.png" alt="Local Hosting Support" style="width:90%" />
+</p>
+
+- An issue is reported view UI or directly
+- A ticket is created in helpdesk app automatically or support creates it if the issue is reported via email/chat.
+- When a new ticket arrives,
+    * A support team member performs the initial investigation
+    * If more information is required to resolve the issue the user is contacted for more information.
+    * If the ticket fits P1/P2 criteria, follow P1/P2 process
+    * Support team member updates the fields of the ticket such as priority, status and assignee.
+    * Depending on the ticket, the support team member might resolve and respond back to the user or escalate it to a different team member from the program or developer team.
+    * If the ticket priority is low, the team might put it into a backlog that can be reviewed later.
+
+[//]: # (    * If the team is not able to get resolve, the ticket can be reported to Dimagi support directly if the team has a support plan or else to the public CommCare developers forum)
+    
+- Once the resolution is found the support team member sends the resolution to the user and closes the ticket after updating relevant ticket fields.
+
+Apart from this a regular periodical (weekly or biweekly) team calls could also be used to coordinate the overall support activities. 
+
+<h5 id="p1-p2-process">
```

Comment:
I'm not a big fan of adding HTML syntax here, but I couldn't get it to link correctly using the native markdown syntax...